source/_posts/2016-02-15-multi-room-audio-with-snapcast.markdown

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title: "Multi-room audio with Snapcast, Mopidy, and Home Assistant"
+date: 2016-02-15 17:10:56 -0500
```

Comment:
- The date has to be updated to when we're going to merge it.
 - You removed the key `date_formatted: "February 13, 2016"` so there is no date rendered on your blog post